src/Form/Admin/TechType.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Form\Admin;
+
+use App\Entity\Tech;
+use App\Enum\TechTypeEnum;
+use App\Form\CategoriesAutocompleteField;
+use App\Form\TechAutocompleteField;
+use App\Form\TechLinksType;
+use App\Form\TechPictureType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\EnumType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class TechType extends AbstractType
```

Comment:
```suggestion
final class TechType extends AbstractType
```

Pareil pour toutes les classes type